src/index.html

```diff
@@ -8,7 +8,45 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="content">
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo image">
+      </a>
+      <nav class="menu">
+        <ul class="menu_list">
+          <li class="nav_item">
+            <a href="#apple" class="nav_link is_active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#samsung" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#smartphones" class="nav_link">Smartphones</a>
+          </li>
```

Comment:
please add empty line between siblings
```suggestion
          <li class="nav_item">
            <a href="#apple" class="nav_link is_active">Apple</a>
          </li>
          
          <li class="nav_item">
            <a href="#samsung" class="nav_link">Samsung</a>
          </li>
          
          <li class="nav_item">
            <a href="#smartphones" class="nav_link">Smartphones</a>
          </li>
```